pages/linux/firewall-cmd.md

```diff
@@ -18,6 +22,10 @@
 
 `firewall-cmd --permanent --zone={{public}} --remove-service={{http}}`
 
+- Permanently open two arbitrary ports in the specified zone (Minecraft service):
```

Comment:
What do you mean by `Minecraft service`?

Reply:
I wanted to provide an example of a service that needs both UDP and TCP protocol, thought it's popular and fun. Let's find a different one then.